### PR TITLE
Feat(sort investigation table) - Add sort options to investigation table

### PR DESCRIPTION
**Pull request by nevobonneh**
new feature [444](https://dev.azure.com/spectrumFactory/CoronaI/_workitems/edit/444/)

### CODE REVIEW

**Review by yaniryos — APPROVED** [review]

File: client/src/components/App/Content/LandingPage/InvestigationTable/InvestigationTable.tsx [hunk 1]
```diff
@@ -26,10 +26,13 @@ const InvestigationTable: React.FC = (): JSX.Element => {
 
     const [selectedRow, setSelectedRow] = React.useState<number>(UNDEFINED_ROW);
     const [investigator, setInvestigator] = React.useState<Investigator>(defaultInvestigator);
-
+    const [order, setOrder] = React.useState<Order>('asc');
```
Comment (NaamanTsur) [inline client/src/components/App/Content/LandingPage/InvestigationTable/InvestigationTable.tsx:29]: You should place the 'asc' and 'dsc' values on an enum

File: client/src/components/App/Content/LandingPage/InvestigationTable/InvestigationTable.tsx [hunk 2]
```diff
@@ -50,10 +61,17 @@ const InvestigationTable: React.FC = (): JSX.Element => {
...
+                                        <TableCell 
+                                        className={key === TableHeadersNames.investigatorName ? classes.columnBorder : ''}
+                                        sortDirection={orderBy === key ? order : false}>
                                             {
                                                 TableHeaders[key as keyof typeof TableHeadersNames]
                                             }
+                                            {sortableCols[key as keyof typeof TableHeadersNames] && <TableSortLabel 
+                                                active
+                                                direction={orderBy === key ? order : 'asc'}
+                                                onClick={(e)=>handleRequestSort(e,key)}>
```
Comment (ItayBenMoshe) [inline client/src/components/App/Content/LandingPage/InvestigationTable/InvestigationTable.tsx:73]: space after ) and =>

**Review by NaamanTsur — CHANGES_REQUESTED** [review]

File: client/src/components/App/Content/LandingPage/InvestigationTable/InvestigationTable.tsx (hunk 2 above)
Comment (nevobonneh) [reply client/src/components/App/Content/LandingPage/InvestigationTable/InvestigationTable.tsx]: fixed

File: client/src/components/App/Content/LandingPage/InvestigationTable/InvestigationTable.tsx (hunk 1 above)
Comment (nevobonneh) [reply client/src/components/App/Content/LandingPage/InvestigationTable/InvestigationTable.tsx]: done

**Review by ItayBenMoshe — APPROVED** [review]

**Review by NaamanTsur — APPROVED** [review]

**Pull request merged**